src/sardana/spock/spockms.py

```diff
@@ -38,19 +38,21 @@
 from sardana.sardanautils import is_pure_str, is_non_str_seq
 from sardana.spock import genutils
 from sardana.util.parser import ParamParser
-from sardana.spock.inputhandler import SpockInputHandler, InputHandler
 from sardana import sardanacustomsettings
 
 CHANGE_EVTS = TaurusEventType.Change, TaurusEventType.Periodic
 
 
 if genutils.get_gui_mode() == 'qt':
+    from taurus.external.qt import Qt
```

Comment:
I think this import is not even necessary.

Reply:
This import was put somewhere else in the same file with comment that Qt needs to be imported ASAP. I figured out that it would good place to keep this import in case it is required. However, I can delete it if you want.

Reply:
This import is currently here:

https://github.com/sardana-org/sardana/blob/f2f73ac20b1f397450109a582e264577cfabd011/src/sardana/spock/spockms.py#L557

and I don't see any comment around it explaining the urgency of the import.

From where you placed it, the subsequent line also imports Qt:
```python
from sardana.taurus.qt.qtcore.tango.sardana.macroserver import QDoor, QMacroServer
```

So, let's try to avoid the import. I hope it won't break anything:)

Reply:
Okay, will do

Reply:
I haven't checked this precise case, but usually the "urgency" for importing Qt from `taurus.external.qt` is related to avoiding indirect imports of PyQt4 API1 bindings by other modules  (e.g. pyqtgraph) which would break taurus.